### PR TITLE
arm64: dts: qcom: msm8916-wingtech-wt86528: add sound

### PR DESCRIPTION
**Pull request by jenneron**

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts [hunk 1]
```diff
@@ -230,6 +308,13 @@
 	extcon = <&usb_id>;
 };
 
+&wcd_codec {
+	qcom,micbias-lvl = <2800>;
+	qcom,mbhc-vthreshold-low = <87 255 312 375 425>;
+	qcom,mbhc-vthreshold-high = <87 237 350 400 450>;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts:314]: Do these thresholds work better than [qcom's defaults](https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/b20608408caff817ec874f325127b07609fbaeb8/sound/soc/msm/msm8x16.c#L1738-L1747)?

Comment (jenneron) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts:314]: seems they both don't always detect SW_MICROPHONE_INSERT. should i change them?

Comment (Minecrell) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts:314]: For now I would prefer using the qcom defaults, unless you know better. Not sure either but I think these are not device specific.

Comment (jenneron) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts]: I changed them to defaults in recent push

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts [hunk 2]
```diff
@@ -230,6 +308,13 @@
 	extcon = <&usb_id>;
 };
 
+&wcd_codec {
+	qcom,micbias-lvl = <2800>;
+	qcom,mbhc-vthreshold-low = <75 150 237 450 500>;
+	qcom,mbhc-vthreshold-high = <75 150 237 450 500>;
+	qcom,hphl-jack-type-normally-open;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86528.dts:315]: There is `qcom,msm-micbias1-ext-cap` in downstream, can you add the equivalent property for mainline here?

**Review by Minecrell — APPROVED** [review]

**Review by Minecrell — APPROVED** [review]

**Pull request merged**